content/docs/06-integrations/00-kubernetes-dashboard.md

```diff
@@ -17,92 +17,604 @@ import Tooltip from "shared/components/ui/Tooltip";
+| --- | --- | --- | --- |
+| `k8s-dashboard.namespace` | | kubernetes-dashboard | The namespace where you want to run the Kubernetes Dashboard deployment |
+| `k8s-dashboard.clusterRole` | Any valid clusterRole from the Kubernetes cluster | `k8s-dashboard-readonly` | The ClusterRole to be attached to the ServiceAccount which  defines RBAC to the cluster resources.<br /><br />By default, a ClusterRole (k8s-dashboard-readonly) with ReadOnly access to most of the resources is deployed. |
+| `k8s-dashboard.certDuration` | A Go time.Duration string format in s (seconds), m (minutes), and h (hour) suffixes | `8760h` (365 days) | Validity for the Self-signed certificate, specified in hours. |
+| `k8s-dashboard.certRenewal` | A Go time.Duration string format in s (seconds), m (minutes), and h (hour) suffixes | `720h` (30 days) | Certificate renew before expiration duration |
+| `k8s-dashboard.serviceType` | ClusterIP, LoadBalancer | ClusterIP | The ServiceType for dashboard. <WarningBox>It is highly recommended to use ClusterIP service type to restrict access to the cluster</WarningBox> |
+| `k8s-dashboard.skipLogin` | True, False | False | A flag to skip authentications in the Kubernetes Dashboard UI. <WarningBox> Enabling this might expose a security risk. Use this only for demo purposes.</WarningBox> |
+
+<WarningBox>
 
+Starting with Kubernetes Dashboard version 2.7.0, the **Connect** button is no longer available. For an optimized experience, we recommend you use the pre-configured version of this dashboard, called Spectro Kubernetes Dashboard pack. To learn more about it and start using it, check out the [Spectro Kubernetes Dashboard](/integrations/spectro-k8s-dashboard) guide. 
+
+</WarningBox>
+
+## Usage
+
+To use the Kubernetes Dashboard pack, you have to add it to your cluster profile. Use the following information to find the Kubernetes Dashboard pack.
+- **Pack Type**: Monitoring
+- **Registry**: Public Repo
+- **Pack Name**: Kubernetes Dashboard
+- **Pack Version**: 2.0.x or higher
+
+The Kubernetes Dashboard pack requires the [Spectro Proxy](/integrations/frp) pack, which serves as a reverse proxy to expose the Kubernetes dashboard. You must configure the Spectro Proxy pack.
+
+
+### Access Kubernetes Dashboard
+
+When connected to the cluster remotely, run the following command to establish a connection to deploy the Kubernetes Dashboard on port 8080. 
 
 <br />
 
+```bash
+kubectl port-forward -n kubernetes-dashboard service/kubernetes-dashboard 8080:443
+```
 
+To access Kubernetes Dashboard, navigate to `https://localhost:8080` in a browser of your choice.
 
-## Versions Supported
+From the Dashboard login page, run the following command from the terminal window to obtain the bearer token:
 
-<Tabs>
+<br />
 
-<Tabs.TabPane tab="2.7.x" key="2.7.x">
+```bash
+kubectl -n kubernetes-dashboard describe secret $(kubectl -n kubernetes-dashboard get secret | grep kubernetes-dashboard-token | awk '{print $1}')

```

Comment:
```suggestion
kubectl --namesoace kubernetes-dashboard describe secret  \
$(kubectl -namespace kubernetes-dashboard get secret | grep kubernetes-dashboard-token | awk '{print $1}')
```
This makes the command easier to ready by placing it into two lines. Just make sure I didn't break the markdown 😓

Reply:
@karl-cardenas-coding Thanks for suggesting this technique. Also for reminding me about using the long form of command options.